lib/phoenix/endpoint/cowboy2_handler.ex

```diff
@@ -1,183 +0,0 @@
-defmodule Phoenix.Endpoint.Cowboy2Handler do
```

Comment:
The majority of this functionality has moved to Plug.Cowboy in https://github.com/elixir-plug/plug_cowboy/pull/88

The reload catch & retry is still pending before this PR is ready for proper review - it can easily land in Endpoint, I think.

Again, not a blocking concern for other work on https://github.com/phoenixframework/phoenix/issues/5003